### DISCUSSION TITLE
Coming Soon

### DISCUSSION THREAD

**Discussion by mcguirepr89** [mcguirepr89/BirdNET-Pi#31] [Show and tell]
# phpSysInfo
![Screenshot from 2021-10-15 09-30-46](https://user-images.githubusercontent.com/60325264/137495304-425a172d-a928-4e2a-8b8a-fdeb5675b92f.png)
![Screenshot from 2021-10-15 09-31-12](https://user-images.githubusercontent.com/60325264/137495260-769f4377-79f9-47c0-bd9a-a0abb530f188.png)
![Screenshot from 2021-10-15 09-31-16](https://user-images.githubusercontent.com/60325264/137495264-60826436-4ca4-41c0-8cf3-52c6b25a2850.png)
![Screenshot from 2021-10-15 09-31-21](https://user-images.githubusercontent.com/60325264/137495265-b9e811f5-5f56-4ee9-98c7-5dae599f31f3.png)

**Comment by mcguirepr89** [comment]
# Update BirdNET-Pi
![Screenshot from 2021-10-15 09-33-10](https://user-images.githubusercontent.com/60325264/137495522-6f7cbcd4-6ba7-407d-b29d-1fe967fbd3ae.png)

**Reply by nilspupils** [reply]
Looking good! May i ask for one more button to stop BirdNET (recording, analysis and extraction)?
Will try the new update as soon as it is ready. 
Best, Nils

**Reply by mcguirepr89** [reply]
![Screenshot from 2021-10-15 15-04-18](https://user-images.githubusercontent.com/60325264/137539604-0aef3beb-478a-49c6-b9f6-4ee2af17307c.png)
It works and will be available soon

**Reply by nilspupils** [reply]
That was faster than light! Top!

**Comment by mcguirepr89** [comment]
[Headless installation guide](https://github.com/mcguirepr89/BirdNET-Pi/wiki/%22Headless%22-installation-using-VNC)

**Comment by mcguirepr89** [comment]
![Screenshot from 2021-10-20 16-45-44](https://user-images.githubusercontent.com/60325264/138169730-58d1e296-4ba8-49ef-a246-4167a8eec05b.png)
New BirdNET-Pi Homepage thanks to @Christoph-Lauer

**Comment by mcguirepr89** [comment]
Tested on iPhone and laptop.
@nilspupils @DD4WH  @CaiusX @Christoph-Lauer -- 
As requested, the installation will soon support creating a HotSpot called "BirdNET-Pi" whenever it is unable to connect to the WiFi you used to install the system. That Access Point auto-starts on boot and once connected, you can access http://birdnetpi.local on your phone (or whatever device does the connecting). On my phone that has an LTE connection (iPhone), because the WiFi from the BirdNET-Pi had no internet access, it retained LTE connectivity also. I'm betting that the Pi fetches the network time from the LTE connection, but I haven't tested that theory yet. 

Take a look at [this repo](https://github.com/0unknwn/auto-hotspot) if you're interested in how it works on the Pi, and [here](https://wiki.archlinux.org/title/wpa_supplicant#wpa_cli_action_script) for how wpa_supplicant works and can be used.

I am going to be testing theoretical field-like scenarios for you guys soon, so if you care to let me know your prospective field process with the new BirdNET-Pi hotspot in mind, please let me know in comments below or in a dedicated discussion.

My best regards,
Patrick

**Reply by nilspupils** [reply]
Any news on this? Or is it already implemented? No rush though as i will only need it in spring... ;)

**Comment by mcguirepr89** [comment]
Color-branded phpsysinfo
![Screenshot from 2021-10-29 20-57-08](https://user-images.githubusercontent.com/60325264/139515268-c06f715a-42ed-4a4f-a34d-70183b322f3d.png)

**Comment by mcguirepr89** [comment]
A BirdNET-Pi Raspberry Pi Image (.img)

- The whole system is installed and running on boot.
- Utilize the new `birdnet-pi-config` tool to change locale (if not _en_US_), default passwords and set lat and longitude (though these values are predicted for you).

**Reply by nilspupils** [reply]
Cool! Great work as always Patrick! Where can i find it?

**Reply by mcguirepr89** [reply]
There are two version right now, but both are _quite_ buggy.

I am hosting the images at https://downloads.pmcgui.xyz

I will announce on the testing discussion when it is time to test the image, though if you like tinkering and linux troubleshooting, MOST of the latest image works :)

**Comment by mcguirepr89** [comment]
Bullseye Pre-Installed Image -- new BirdNET-Pi Configuration GUI

Walk-through setup, with option for offline configuration.
Posting soon

**Comment by mcguirepr89** [comment]
![image](https://user-images.githubusercontent.com/60325264/149671187-0a66c0d6-d5de-4851-823e-346aff360499.png)
![image](https://user-images.githubusercontent.com/60325264/149672259-0dde8c45-b555-4e70-8b72-b3340b154776.png)

**Reply by mcguirepr89** [reply]
@timsterc -- I'm working on the PHP forms I had mentioned not long ago. This is just a very preliminary preview.

**Comment by mcguirepr89** [comment]
![Screenshot 2022-05-13 at 10 21 55 AM](https://user-images.githubusercontent.com/60325264/168304979-ea481396-13a6-45aa-a20f-85d1859d2c98.png)
Custom Captive Portal for Field Connectivity and pre-installed image easy setup

**Reply by timsterc** [reply]
@mcguirepr89 - that's very cool!

**Reply by ehpersonal38** [reply]
Such ease of use!!

**Comment by mcguirepr89** [comment]
Rclone backups -- full web interface for configuring regular backups from rclone
![image](https://user-images.githubusercontent.com/60325264/176699355-b41f9a06-a629-476f-9d3d-32363b2e9e28.png)
